src/components/Requirements/SubRequirement.vue

```diff
@@ -28,47 +36,47 @@
           >
         </p>
       </div>
-      <div v-if="displayDescription" :class="[{ 'completed-ptext': isFulfilled }, 'description']">
-        {{ subReq.requirement.description }}
-        <a
-          class="more"
-          :style="{ color: `#${color}` }"
-          :href="subReq.requirement.source"
-          target="_blank"
-        >
-          <strong>Learn More</strong></a
-        >
-        <div v-if="subReq.requirement.fulfilledBy === 'toggleable'">
-          <div class="toggleable-requirements-select-wrapper">
+    </button>
+    <div v-if="displayDescription" :class="[{ 'completed-ptext': isFulfilled }, 'description']">
+      {{ subReq.requirement.description }}
+      <a
+        class="more"
+        :style="{ color: `#${color}` }"
+        :href="subReq.requirement.source"
+        target="_blank"
+      >
+        <strong>Learn More</strong></a
+      >
+      <div v-if="subReq.requirement.fulfilledBy === 'toggleable'">
+        <div class="toggleable-requirements-select-wrapper">
+          <div
+            class="toggleable-requirements-select toggleable-requirements-input"
+            v-click-outside="closeMenuIfOpen"
+          >
             <div
-              class="toggleable-requirements-select toggleable-requirements-input"
-              v-click-outside="closeMenuIfOpen"
+              class="toggleable-requirements-dropdown-placeholder toggleable-requirements-dropdown-wrapper"
+              @click="showFulfillmentOptionsDropdown = !showFulfillmentOptionsDropdown"
```

Comment:
Just wanted to note that I'm unable to tab to see and select the fulfillment option. That could be done in another PR since this one is getting big.

https://user-images.githubusercontent.com/54298311/109200653-dd6dca80-776e-11eb-9f8b-dc7029cda6f7.mov

Reply:
ill address all the other dropdowns in a future PR!